content/messaging/agent-chat-api/v3.2/index.mdx

```diff
@@ -1369,10 +1375,9 @@ curl -X POST \
       "is_followed": false
     }
   ],
-  "pagination": {
-    "page": 1,
-    "total": 1
-  }
+  "found_chats": 42,
+  "next_page_id": "MTUxNzM5ODEzMTQ5Ng==", // optional
+  "previous_page_id": "MTUxNzM5ODEzMTQ5Ng==" // optional
```

Comment:
```suggestion
  "previous_page_id": "MTUxNzM5ODEzMTQ5Ng=="
```

Reply:
I suggest we avoid those "JSON comments" when possible. I'll be soon moving all the responses to individual JSON files and will need to delete the "comments" anyway.